### PR TITLE
improve dev workflow

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: deploy/olm-catalog/ibm-catalog-ui-operator/3.6.2/ibm-catalog-ui-operator.v3.6.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,9 +8,6 @@ metadata:
           "apiVersion": "operator.ibm.com/v1alpha1",
           "kind": "CatalogUI",
           "metadata": {
-            "annotations": {
```
Comment (bitscuit) [inline deploy/olm-catalog/ibm-catalog-ui-operator/3.6.2/ibm-catalog-ui-operator.v3.6.2.clusterserviceversion.yaml:11]: v3.6.2 already released. Need to bump version and then make changes

File: deploy/operator.yaml [hunk 2]
```diff
@@ -57,7 +57,7 @@ spec:
             - name: OPERATOR_NAME
               value: "ibm-catalog-ui-operator"
             - name: OPERAND_CATALOG_UI_IMAGE
-              value: "quay.io/opencloudio/icp-catalog-ui@sha256:dbf31453dbb943a1ddd5a8d72fa89f3136232039f91e990148fdc5a060b75332"
```
Comment (bitscuit) [inline deploy/operator.yaml:60]: is this the next digest that will be pushed to quay?

File: deploy/olm-catalog/ibm-catalog-ui-operator/3.6.2/ibm-catalog-ui-operator.v3.6.2.clusterserviceversion.yaml (hunk 1 above)
Comment (cheewaio) [reply deploy/olm-catalog/ibm-catalog-ui-operator/3.6.2/ibm-catalog-ui-operator.v3.6.2.clusterserviceversion.yaml:11]: No, v3.6.1 was the last release on stable-v1
https://github.com/IBM/ibm-catalog-ui-operator/blob/master/deploy/olm-catalog/ibm-catalog-ui-operator/ibm-catalog-ui-operator.package.yaml#L6
https://github.ibm.com/IBMPrivateCloud/ibm-catalog-ui-operator-case/blob/master/stable/ibm-catalog-ui-operator-bundle/case/ibm-catalog-ui-operator/inventory/ibmCatalogUiOperatorSetup/resources.yaml#L43

File: deploy/operator.yaml (hunk 2 above)
Comment (cheewaio) [reply deploy/operator.yaml:60]: yes, https://travis.ibm.com/IBMPrivateCloud/catalog-ui/jobs/36351283#L288

**Review by bitscuit — APPROVED** [review]
/lgtm

**Pull request merged**